src/components/Overlay/README.md

```diff
@@ -0,0 +1,32 @@
+<!--GITHUB_BLOCK-->
+
+# Loader
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Overlay} from '@gravity-ui/uikit';
+```
+
+The `Overlay` component renders an overlay over the parent element with relative position.
```

Comment:
it should be highlighted here that the component should be used with a parent with `position: relative`